crates/wasi-provider/src/lib.rs

```diff
@@ -69,34 +66,38 @@ impl Provider for WasiProvider {
         //   - mount any volumes (popen)
         //   - run it to completion
         //   - bail if it errors
-        info!("Starting containers for pod {:?}", pod.name());
+        let pod_name = pod.name().to_owned();
```

Comment:
Is to_owned needed? Is this to get around borrowing?

Reply:
Yes. This was because we need to move the pod into the new `PodHandle` further down